action/action.go

```diff
@@ -110,13 +109,24 @@ func createMetadataJson(in input) string {
 	}
 	actions.Infof("Working sha %v\n", sha)
 
+	// FIXME: If we are using implicit values for repository ($GITHUB_REPOSITORY), do we even need the org?
 	org := in.org
 	if org == "" {
-		org = defaultRepositoryOwner
-	}
-	repository := in.repo
-	if repository == "" {
-		repository = strings.Split(os.Getenv("GITHUB_REPOSITORY"), "/")[1]
+		actions.Fatalf("organization is empty")
+	}
+
+	// The repository may be in one of two formats.
```

Comment:
In Github's response body, this is called `repository.full_name` but in other systems I've seen it called the repo "slug". With clear enough descriptions in the actions input, we should be able to get the users to put in either the `org/repo` and parse or `org` and `repo` fields respectively. To make the least amount of changes we can go with the latter and update the descriptions with an example of what we expect.